src/main/java/Implementations/ComputationImpl.java

```diff
@@ -40,27 +48,20 @@ public long[][] performDigitFactorial(String[] numberStrings) {
                 results[0][i] = Long.parseLong(numberStrings[i]);
                 results[1][i] = digitFactorialSum(numberStrings[i]);
             } catch (NumberFormatException e) {
-                System.err.println("Invalid number format " + e.getMessage());
+                System.err.println("Invalid number format: " + e.getMessage());
                 results[0][i] = 0;
                 results[1][i] = -1;
             }
         }
         return results;
     }
 
-    private static long factorial(int n) {
-        long result = 1;
-        for (int i = 2; i <= n; i++) {
-            result *= i;
-        }
-        return result;
-    }
-
     private static long digitFactorialSum(String numberString) {
```

Comment:
Simple and effective implementation of using the array to perform a factorial on number 0-9.